python/lsst/pipe/tasks/calexpCutout.py

```diff
@@ -0,0 +1,96 @@
+    )
+    cutouts = pipeBase.connectionTypes.Output(
+        doc="Cutouts",
+        name="calexp_cutouts",
+        storageClass="Stamps",
+        dimensions=("instrument", 'visit', 'detector'),
+    )
+
+
+class CalexpCutoutTaskConfig(pipeBase.PipelineTaskConfig,
+                             pipelineConnections=CalexpCutoutTaskConnections):
+    """Configuration for CalexpCutoutTask
+    """
+    max_cutouts = Field(dtype=int, default=100, doc='Maximum number of entries to process. '
+                                                    'The result will be the first N in the catalog.')
+    skip_bad = Field(dtype=bool, default=True, doc='Skip cutouts that do not fall completely within'
+                                                   ' the calexp bounding box?')
+
+
+class CalexpCutoutTask(pipeBase.PipelineTask):
+    """Task for computing cutouts on a specific calexp given
+    positions and sizes of the stamps.
+    """
+    ConfigClass = CalexpCutoutTaskConfig
+    _DefaultName = "calexpCutoutTask"
+
+    def run(self, in_cat, calexp):
+        """Compute and return the cutouts.
+
+        Parameters
+        ----------
+        in_cat : `dict`
+            A dictionary containing at least the following columns: ra, dec, size.
+            The coordinates should be in ICRS degrees.  The size is in pixels.
+        calexp : `lsst.afw.image.ExposureF`
+            The calibrated exposure from which to extract cutouts
+
+        Returns
+        -------
+        output : `lsst.pipe.base.Struct`
+            A struct containing a container class that wraps a list of
+            masked images of the cutouts and a PropertyList containing
+            the metadata to be persisted with the cutouts
+        """
+        max_idx = self.config.max_cutouts
+        cutout_list = []
+        wcs = calexp.getWcs()
+        mim = calexp.getMaskedImage()
+        metadata = calexp.getMetadata()
+        metadata['RA_DEG'] = in_cat['ra'][:max_idx]
+        metadata['DEC_DEG'] = in_cat['dec'][:max_idx]
+        metadata['SIZE'] = in_cat['size'][:max_idx]
+        for ra, dec, size in zip(in_cat['ra'][:max_idx], in_cat['dec'][:max_idx],
+                                 in_cat['size'][:max_idx]):
+            pt = geom.SpherePoint(geom.Angle(ra, geom.degrees),
+                                  geom.Angle(dec, geom.degrees))
+            pix = wcs.skyToPixel(pt)
+            # Clamp to LL corner of the LL pixel and draw extent from there
+            box = geom.Box2I(geom.Point2I(int(pix.x-size/2), int(pix.y-size/2)),
+                             geom.Extent2I(size, size))
+            if not mim.getBBox().contains(box):
+                if not self.config.skip_bad:
+                    raise ValueError(f'Cutout bounding box is not completely contained in the image: {box}')
+                else:
+                    continue
+            # I think we need to think about what we want the origin to be: LOCAL or PARENT
```

Comment:
Please file a ticket on this subject and reference the ticket number here.

But when in doubt: I suggest parent. It's more informative.

Reply:
I just went with the default.  I'm happy to change if you think it would be better

Reply:
I have no idea what the default is. I think PARENT would be best. If that's the default then fine. Or if you have a good reason to prefer LOCAL. 

Given the ambiguity I suggest documenting your choice in the doc string for this task!